posthog/temporal/data_imports/sources/zendesk/source.py

```diff
@@ -102,3 +102,13 @@ def source_for_pipeline(self, config: ZendeskSourceConfig, inputs: SourceInputs)
                 else None,
             )
         )
+
+        # assumes datetime as all incremental zendesk fields are datetime based
+        if inputs.should_use_incremental_field and inputs.incremental_field:
+            zendesk_source_response.partition_count = 1
+            zendesk_source_response.partition_size = 1
+            zendesk_source_response.partition_mode = "datetime"
+            zendesk_source_response.partition_format = "week"
+            zendesk_source_response.partition_keys = [inputs.incremental_field]
```

Comment:
We can't use generated timestamp here - the value changes. 

> The generated_timestamp value is updated for all ticket updates, including system updates. If a system update occurs after a ticket event, the unchanged updated_at time will become earlier relative to the updated generated_timestamp time.

From https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/

Tickets have a created at field